kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -6754,7 +6754,7 @@ def make_vnc():
             else:
                 vnc = e(devices, 'graphics', None,
                         {'type': 'vnc', 'port': '5900', 'autoport': 'yes', 'passwd': str(cmd.consolePassword)})
-            e(vnc, "listen", None, {'type': 'address', 'address': '0.0.0.0'})
+            e(vnc, "listen", None, {'type': 'address', 'address': getattr(cmd, 'vncListenAddress', None) or '0.0.0.0'})
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)